web_hierarchy/static/src/xml/web_hierarchy_template.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<template>
+     <t t-name="HierarchyView">
+       <div id="chart_div" style="margin: 50px 0 0 0;">
```

Comment:
Consider not using inline styles and putting these in a css file (which you already have)